templates/docker-monolithic/docker-compose.override.yml

```diff
@@ -2,4 +2,4 @@ version: '3.8'
 
 services:
   supportpal:
-    hostname: 'supportpal.example.com'
+    hostname: ''
```

Comment:
The only reason this is kept is to provide a valid compose file. This should be changed with an upcoming pr (that allows ports customization)

Reply:
What do you mean "valid compose file"? It's valid without the `hostname` directive